### DISCUSSION TITLE
How to use an existing Azure Container Registry

### DISCUSSION THREAD

**Discussion by nomocas** [nitrictech/nitric#638] [Q&A] [answered]
Hi All,

Is it possible to specify which ACR to use when deploying on Azure?

(Of course, I checked everywhere if there is any information on that, and even in the source code ;))

Cheers.

**Comment by jyecusch** [comment, marked as answer]
Hi @nomocas the standard Azure providers don't currently have this configuration option. That said I think it would be easy enough to add. We'd be open to a contribution if you're interested, otherwise I'm happy to create a feature request from this post and we can look into it.

Another option would be to customize your own provider to add this for your use.
